src/actions/integrations.js

```diff
@@ -134,3 +134,36 @@ export function updateOutgoingHook(hook) {
         hook
     );
 }
+
+export function addCommand(teamId, command) {
+    return bindClientFunc(
+        Client.addCommand,
```

Comment:
Please use `Client4`, we're moving away from API v3 which is what `Client` uses. You might need to add the functions into `src/client/client4.js` for any missing ones you need. You can use https://api.mattermost.com/v4 for reference